docs/volume_annotation.md

```diff
@@ -92,6 +94,17 @@ Similar to the above interpolation feature, you can also extrude the currently a
 This means, that you can label a segment on one slice (e.g., z=10), move a few slices forward (e.g., z=12) and copy the segment to the relevant slices (e.g., z=11, z=12).
 The extrusion can be triggered by using the extrude button in the toolbar (also available as a dropdown next to the interpolation/extrusion button).
 
+### Volume Flood Fills
```

Comment:
Interesting diff. I move the section on volume interpolation up and above flood fills since it is more relevant.